doc/manuals/deprecated.md

```diff
@@ -30,3 +30,5 @@ A list of deprecated features and the version in which they were deprecated foll
     the broker runs in "ConfMan mode".
 * Configuration Manager role (deprecated in 0.21.0, removed in 0.25.0)
 * Associations (deprecated in 0.21.0, removed in 0.25.0).
+* Deprecated command line argument in Orion 0.27.0:
```

Comment:
Move it to the first place (i.e. before the bullet taking about ONTIMEINTERVAL). The list follows a cronological-inverse ordering.